pos_blstm/create_data.py

```diff
@@ -0,0 +1,124 @@
+from gensim import models
+import pickle
+import utils
+import pdb
+import numpy as np
+import pdb
+
+model = models.Word2Vec.load('tmp/embedding_map')
```

Comment:
Definitely going to remove gensim later, but a good word embedding is going to necessary in order to get the best accuracy. Just using Word2Vec for now to build the BLSTM.